server/aws-lsp-codewhisperer/src/language-server/inline-completion/codeWhispererServer.test.ts

```diff
@@ -73,7 +78,7 @@ const startServer = async (features: TestFeatures, server: Server): Promise<Test
     return features
 }
 
-describe('CodeWhisperer Server', () => {
+describe.only('CodeWhisperer Server', () => {
```

Comment:
wait why do we need `describe.only` here?

Reply:
ooops I shouldn't commit that, will revert, thanks for catching!